Feat list translation and list view scrolling

rummens opened the pull request:
## List Blueprint

- Added translation for German, Spanish and French.
- Added new option to only speak out a certain number of items on the shopping list
- Added new option to overwrite the name of the shopping list (Mine has a very long name as I use an integration with Thermomix and cannot change the name of the list)
- Added proposal for new README structure and updated wiki accordingly.

dinki commented:
Nice work.  Cool improvements for sure.  Note that the list should remain as a single column but if you'd like to add multicolumn that can be created as an alternative view (similar to clock).  I want to keep the default as a single column but the ability to scroll is definitely beneficial!

dinki commented:
Also, if you can, please keep the PRs focused on one thing and don't bundle additional.  Make a separate for timer mods you are planning.  There may be an easy way to separate things but I'm still a novice and would prefer that if possible unless there is some sort of tie in.

rummens commented:
Makes sense. I removed the failed attempt to add columns, updated the wiki and will keep this PR focused on the list things. With that I think this is good to merge.

dinki commented:
Okay this is looking pretty great.  A few observations and hopefully things you can modify.

I'm seeing your .gitignore coming through in the PR.  Please remove.

![image](https://github.com/user-attachments/assets/cb7d3f69-1eda-4876-9cb4-63dca48910a2)
Before
![image](https://github.com/user-attachments/assets/0b9e0879-8bea-4cab-98fa-db1f450f444d)
After

Can you please look at replicating the size and weight?   Options can be added to modify these sizes if necessary but please remember that we are looking for something like a 5' or 10' interface so that people can read these while not next to their displays.  My goal is not to ever touch my device but to have it visible from across the room.  Understood others prefer the close interaction.

When adding an item I am getting an AI response.  It does add the item but it does not display the view.  Instead it shows the AI view.  I may need to check this closer on my install.  Can you confirm that your BP is picking up these requests locally and that it displays the list view when adding?

I am seeing multiple items with the same name but different cases (ie 'Crackers' and 'crackers').  I am thinking that this is due to how the STT is returning.  Please look at this and perhaps adjust the values to be capitalized (ie  {{ variable | captitalize }} and then do the comparison.

I noticed you used a different method for grouping translations.  Perhaps your way is better but the existing BPs use this format:

```
      translations:
        en:
          responses:
            time: "It's {time} "
            day_of_week: "Today is {day_of_week}"
            date: "Today is {date}"
        de:
          responses:
            time: "Es ist {time} Uhr"
            day_of_week: "Heute ist {day_of_week}"
            date: "Heute ist {date}" 
 ```

I would like consistency for all blueprints.  I'm open to hearing the merits of doing it the way you did it versus the way I am doing it.


And after all that, great work!!  This is very close to done and I definitely appreciate the efforts.  As you know, this one has a lot going on and the help in getting this one complete is fantastic!

Are you on Discord?  If so, what is your username?

rummens commented:
Hi,

thanks for the good feedback. 

I committed the .gitignore on purpose, figured other people might also use an IDE and this protects against accidentally committing metadata files. But if you want I can remove those two lines?

I fixed the font size issue. That was my mistake, I accidentally removed the full CSS block when removing the attempt to add multi column but didn't realise that also removed the font size.

For me the list view is opened when adding/removing things or asking for it. What do you mean with AI view? For me the list view is opened (or whatever is specified in the view var).

I added the enhanced comparison. Really good idea.

I also updated the translation dictionary, honestly didn't really think about the structure. So no feeling for either structure.

I am on discord yes and we already chatted about this :) My handle is marcel280396.

dinki commented:
Lots of work done here!  Wow.  Thank you.  I will look at it soon but may be busy this weekend.  Apologies for not connecting the discord name to your GH name.  I am often distracted trying to do three million things at once!

rummens commented:
No worries. We are not in a rush :-)

dinki commented:
Tested and working great.  I will merge it.  Thanks again.

I will probably circle back and change from radial buttons to pulldown like the 'find my family' blueprint and move language up to top slot but that should be it for this one.

dinki commented:
@rummens I didn't notice but this was submitted to main instead of viewassist-multilingual.  Seems I can't change and remerge.  Can I bother you to resubmit or can you guide me on how to fix?

rummens commented:
Ohh my bad, didn't realise it was suppose to go into a different branch. I see you already reverted the merge. I just added the small tweak to the inputs and Ill submit a new PR now.
